Symfony/CS/Tokenizer/Tokens.php

```diff
@@ -1404,4 +1404,45 @@ public function isMonolithicPhp()
 
         return 0 === count($kinds[T_INLINE_HTML]) + count($hhvmHashBangs) && 1 === count($kinds[T_OPEN_TAG]) + count($kinds[T_OPEN_TAG_WITH_ECHO]) + count($hhvmOpenTagsWithEcho);
     }
+
+    /**
+     * Clear token and merge surrounding whitespace tokens.
+     *
+     * @param int $index
+     */
+    public function clearTokenAndMergeSurroundingWhitespace($index)
+    {
+        $count = count($this);
+        $this[$index]->clear();
+        if ($index === $count - 1) {
+            return;
+        }
+
+        // We are looking for the following sequence:
+        // [w][c](0..N)[index][c](0..N)[w]
```

Comment:
I remember ^-^
this sequence:
[1 white space token][any number of cleared tokens][token at given index][any number of cleared tokens][1 white space token]

not sure why, I'm checking your PR now :)